src/mrnet/core/rates.py

```diff
@@ -41,47 +41,39 @@ class ReactionRateCalculator(MSONable):
 
     def __init__(self, reactants, products, transition_state):
 
-        self.reactants = reactants
-        self.products = products
         self.transition_state = transition_state
 
         # Assume rate law is first-order in terms of each reactant/product
-        rate_law = {
-            "reactants": {ii: 1 for ii in range(len(self.reactants))},
-            "products": {jj: 1 for jj in range(len(self.products))},
+        self.rate_law = {
+            "reactants": np.ones(len(reactants)),
+            "products": np.ones(len(products)),
         }
 
-        self.rate_law = rate_law
+        # Store relevant information from reactants, products, rather than storing obj
+        # Here, they are stored as np arrays, which have a smaller footprint
+        self.pro_energy = sum([p.energy for p in products])
+        self.rct_energy = sum([r.energy for r in reactants])
 
-    @property
-    def net_energy(self):
-        """
-        Calculate net reaction energy (in eV).
-        """
-        rct_energies = [r.energy for r in self.reactants]
-        pro_energies = [p.energy for p in self.products]
+        self.pro_enthalpy = sum([p.enthalpy for p in products])
+        self.rct_enthalpy = sum([r.enthalpy for r in reactants])
 
-        return (sum(pro_energies) - sum(rct_energies)) * 27.2116
+        self.pro_entropy = sum([p.entropy for p in products])
+        self.rct_entropy = sum([r.entropy for r in reactants])
 
-    @property
-    def net_enthalpy(self):
-        """
-        Calculate net reaction enthalpy (in eV).
-        """
-        rct_enthalpies = [r.enthalpy for r in self.reactants]
-        pro_enthalpies = [p.enthalpy for p in self.products]
+        self.rct_free_energy_methods = [r.get_free_energy for r in reactants]
+        self.pro_free_energy_methods = [p.get_free_energy for p in products]
```

Comment:
Don't store the method. Just have a method to calculate free energy from energy, enthalpy, and entropy. Actually, it might be wise to have a free energy function in the utils or something, since we'll want to be able to calculate that in the entries, in the rates, etc.